libs/api/prisma/schema.prisma

```diff
@@ -982,3 +982,59 @@ model TaggedEvents {
   @@id([eventId, tagId])
   @@map("events.tagged-events")
 }
+
+model SubscriptionTimeline {
+	id                  String @id @default(cuid())
```

Comment:
We are in the process of switching to GUID for everything, could we move that to GUID already? the Event model above should have an example.